src/layout/IHyprLayout.cpp

```diff
@@ -338,8 +338,10 @@ void IHyprLayout::onMouseMove(const Vector2D& mousePos) {
     } else if (g_pInputManager->dragMode == MBIND_RESIZE || g_pInputManager->dragMode == MBIND_RESIZE_FORCE_RATIO || g_pInputManager->dragMode == MBIND_RESIZE_BLOCK_RATIO) {
         if (DRAGGINGWINDOW->m_bIsFloating) {
 
-            Vector2D MINSIZE = g_pXWaylandManager->getMinSizeForWindow(DRAGGINGWINDOW).clamp({20, 20});
-            Vector2D MAXSIZE = g_pXWaylandManager->getMaxSizeForWindow(DRAGGINGWINDOW);
+            Vector2D MINSIZE = g_pXWaylandManager->getMinSizeForWindow(DRAGGINGWINDOW)
+                                   .clamp({DRAGGINGWINDOW->m_sAdditionalConfigData.minSize.toUnderlying().x, DRAGGINGWINDOW->m_sAdditionalConfigData.minSize.toUnderlying().y});
```

Comment:
`toUnderlying` will ret a vec, why are you decomposing it and then composing back??